src/app/components/home/home.component.html

```diff
@@ -15,10 +15,9 @@
 
         <ng-container *ngFor="let candidateType of sortedObj | keyvalue: asIsOrder; let i= index">
 
-          <mat-expansion-panel (opened)="setOfficeStep(i)" hideToggle="true" (closed)="setCouncilDistrictStep(-1)">
+          <mat-expansion-panel routerLinkActive #rla="routerLinkActive" [expanded]="officeStep===i || rla.isActive" (opened)="setOfficeStep(i)" hideToggle="true" (closed)="setCouncilDistrictStep(-1)">
 
-            <mat-expansion-panel-header [ngClass]="{'active-link':officeStep===i}" collapsedHeight="50px" expandedHeight="50px" 
-               [routerLink]="candidateType.value.deepTree? []: '/'+ candidateType.key">
+            <mat-expansion-panel-header [ngClass]="{'active-link': officeStep===i || rla.isActive}" collapsedHeight="50px" expandedHeight="50px" [routerLink]="'/'+ candidateType.key">
```

Comment:
why was the ternary operation for routerlink removed, [routerLink]="candidateType.value.deepTree? []: '/'+ candidateType.key"  ? I think for some candidates that have nested children eg:-  city council , the user is not routed to a component when the user clicks on city council.

Reply:
That makes sense. I removed it because for "city council", candidateType.key never populated with anything, but will add it back in to avoid other use cases where `candidateType.key` would output a string.

Is there a reason for outputting an empty array vs an empty string in the case where it has a deepTree?

Reply:
![image](https://user-images.githubusercontent.com/20343877/99919270-3aba3180-2cd1-11eb-858b-6cca4a9259e9.png)

Reply:
we can pass null or undefined and it will be interpreted as empty [] by the directive or we can directly pass an []